### DISCUSSION TITLE
Victron status codes

### DISCUSSION THREAD

**Discussion by drtinaz** [slipx06/sunsynk-power-flow-card#444] [Ideas]
I commented on another discussion but I thought it would be best to open another feature request.
Can you add status codes for victron inverters? Here is a list of status codes that should cover all Vebus inverters.

0=Off
1=Low Power
2=Fault
3=Bulk
4=Absorption
5=Float
6=Storage
7=Equalize
8=Passthru
9=Inverting
10=Power assist
11=Power supply
244=Sustain
252=External control

Thanks for your hard work. Really appreciate the card. Better than anything else I have seen.

**Comment by slipx06** [comment]
Added in PR #448
